adapters/oidc/js/static/3p-cookies-step1.html

```diff
@@ -1,27 +1,26 @@
-<!--
-  ~ Copyright 2020 Red Hat, Inc. and/or its affiliates
-  ~ and other contributors as indicated by the @author tags.
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~ http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
-
 <!DOCTYPE html>
 <html>
-<body>
-<script>
-    document.cookie = "KEYCLOAK_3P_COOKIE_SAMESITE=supported; max-age=60; samesite=none; secure"
-    document.cookie = "KEYCLOAK_3P_COOKIE=supported; max-age=60"
```

Comment:
Not quite sure why we were setting this other cookie before. @vmuzikar perhaps you can shine a light on this?

Reply:
IIRC, this had to do something with the Apple bug around SameSite that treated `SameSite=None` as `Strict`. Additionally, to support scenarios without TLS (`SameSite=None` requires `Secure`).

Reply:
Alright, I am going to add this back to support those edge cases.